events/artsy-learns-to-code-v2.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Artsy Learns to Code v2
+description: Second iteration on ALTC
+---
+
+See the original ALTC write-up [here](./artsy-learns-to-code.md) for the previous iteration's abstract and
+structure.
+
+## Abstract
+
+Let's kick off another session of Artsy Learns to Code!
+
+## Motivations for ALTC v2
+
+- It's social! Lets Artsy people interact across their normal silos
+- Build empathy across Artsy
+- It's fun! Boosts morale
+- Empowers non-technical people
+- Gets engineers used to teaching regularly
```

Comment:
This was especially huge for me last time - I definitely was challenged to understand the tools I use on a daily basis better because I was teaching them to other folks who had the annoying habit of asking good questions